include/GafferImage/Median.h

```diff
@@ -41,12 +41,12 @@
 #include "Gaffer/StringPlug.h"
 #include "Gaffer/TypedObjectPlug.h"
 
-#include "GafferImage/ImageProcessor.h"
+#include "GafferImage/RankFilter.h"
```

Comment:
We can remove the other includes above now, since they're not needed directly by this file.